go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -457,10 +460,6 @@ func (throttler *Throttler) Disable() bool {
 	}
 	log.Infof("Throttler: disabling")
 	// _ = throttler.updateConfig(ctx, false, throttler.MetricsThreshold.Get()) // TODO(shlomi)
-	throttler.aggregatedMetrics.Flush()
-	throttler.recentApps.Flush()
-	throttler.nonLowPriorityAppRequestsThrottled.Flush()
```

Comment:
These are done in the reverse order now because they are in defers. I'm assuming that's fine.

Reply:
Yes, that's fine. Ordering is irrelevant here.